content/blog/understanding-esp32’s-security-features/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Understanding ESP32’s Security Features
+date: 2018-05-31
+showAuthor: false
+authors: 
+  - kedar-sovani
+---
```

Comment:
Could the migration script auto-generate the alias matching the name of the original article (`/understanding-esp32s-security-features-14483e465724`)? (I left a comment about this here: https://github.com/espressif/developer-portal/pull/78#issuecomment-2302148096.)
This way we can later redirect blog.espressif.com domain to developer.espressif.com, and the old links will still work.